pkg/envoy/api/listener.go

```diff
@@ -217,6 +217,9 @@ func createTLSContext(certificate []byte, privateKey []byte) *auth.DownstreamTls
 	return &auth.DownstreamTlsContext{
 		CommonTlsContext: &auth.CommonTlsContext{
 			AlpnProtocols: []string{"h2", "http/1.1"},
+			TlsParams: &auth.TlsParameters{
+				TlsMinimumProtocolVersion: auth.TlsParameters_TLSv1_2,
```

Comment:
Could you add a comment that we don't need this once envoy image is newer than 1.23 as you commented in the description?
> "envoyproxy has adopted this change in https://github.com/envoyproxy/envoy/commit/f8baa480ec9c6cbaa7a9d5433102efb04145cfc8 (v1.23)"

Reply:
Done